### PR TITLE
New teams ui2 - new Videos, Collaborations Admin, Settings pages

### PR DESCRIPTION
**Pull request by PCF-Testing**
Includes new design and features for collab-based teams. Covers Videos, Collaborations Admin, Collaboration Details, Members, Team Settings pages. Tested locally.

### CODE REVIEW

File: templates/editor/workflow-review.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <div ng-switch-when="review">
     <h2>[[ workMode.heading ]]</h2>
 
-    <p>{% trans "Watch the video and verify that the subtitles are correct. Click Approve when done, or Save Draft to finish later" %}<br>
+    <p>[[ workMode.helpText ]]</h2>
```
Comment (syl22-00) [inline templates/editor/workflow-review.html]: Seems like a mismatch in HTML tags, will fix it after the merge.

File: templates/new-teams/base.html [hunk 2]
```diff
@@ -52,12 +53,16 @@
...
-        {% for page in team.new_workflow.extra_settings_pages %}
+      {% for page in team|extra_settings_pages:request.user %}
         <li>
           <a href="{{ page.url }}">{{ page.title }}</a>
         </li>
         {% endfor %}
         <li>
+          <a href="{% url "teams:settings_feeds" slug=team.slug %}">{% trans "Video Feeds" %}</a>
+        </li>
+        <li>
```
Comment (syl22-00) [inline templates/new-teams/base.html]: Might be an extra `<li>` tag

File: templates/new-teams/members.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    {% endfor %}
+    </ul>
+    {% else %}
+    <p class="empty">{% trans "No members found" %}</p>
+    {% endif %}
+  </div>
+</div>
+{% if edit_form.editable_member_ids %}
+<aside class="modal" id="edit-form">
+  <h3></h3>
```
Comment (syl22-00) [inline templates/new-teams/members.html]: Is it necessary?

File: templates/new-teams/settings-feeds.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+<aside class="modal" id="confirm-delete">
+  <form method="post" action="">
+    {% csrf_token %}
+    <input type="hidden" name="action" value="delete">
+    <input type="hidden" name="feed">
+
+    <h3></h3>
+    <p class="text"></p>
+
```
Comment (syl22-00) [inline templates/new-teams/settings-feeds.html]: Are those empty fields necessary?

File: templates/new-teams/members.html (hunk 3 above)
Comment (bendk) [inline templates/new-teams/members.html]: In a couple places I had empty tags like this that were filled in by javascript later.

**Pull request merged**